src/test/java/core/basesyntax/operations/FruitActionTest.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax.operations;
+
+import core.basesyntax.service.CsvFileReaderImpl;
+import core.basesyntax.storage.Storage;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FruitActionTest {
+    private static FruitAction act;
+    private static CsvFileReaderImpl fileReader;
+
+    @BeforeClass
+    public static void beforeClass() {
+        act = new FruitAction();
+        fileReader = new CsvFileReaderImpl();
+    }
+
+    @Before
+    public void before() {
+        Storage.getStockBalance().clear();
+    }
+
+    @Test
+    public void actionSupplyTest() {
+        List<List<String>> testList;
+        String path = "src/main/resources/supplyCorrect.csv";
```

Comment:
```suggestion
        String path = "src/test/resources/supplyCorrect.csv";
```
test files should be in the test resource folder